proposals/2161/registry.md

```diff
@@ -28,6 +30,11 @@ Each consumer or producer entity can claim a name, defined in the registry: “{
 Each registered entity will provide a documentation URL for its documentation.
 The registered name is used to shorten “producer” and “schemaUrl” fields in facets.
 
+## Acceptance Guidelines
+To claim a name, an entity must have either documentation or a test/sample. "Reserving" a name prior to public functionality is discouraged.
+(When should one register a new facet, vs. an extension to an existing facet? Example: do I make WarningMessageRunFacet, or do I extend ErrorMessageRunFacet to have a "level" field where I can set it to "ERROR", "WARNING", "DEBUG", etc?)
+(how does one register an extension to an existing facet?)
```

Comment:
We haven't defined that. Maybe just update the existing facet in a backward compatible way?

Reply:
I'll remove the question, and when an extension comes up we can add guidelines then :D